components/date-input/src/DateInput.tsx

```diff
@@ -50,11 +50,15 @@ DateInput.format = (v: string): string => {
     : (
       v.includes('/')
         ? v.split('/')
-        : [
-          v.slice(0, 2),
-          v.slice(2, 4),
-          v.slice(4)
-        ]
+        : (
+          v.includes('.')
+            ? v.split('.')
+            : [
+              v.slice(0, 2),
+              v.slice(2, 4),
+              v.slice(4)
+            ]
+        )
```

Comment:
Nested ternaries are starting to make the code more unreadable. I think starting to create functions to deal with these operations is becoming necessary. This also applies throughout the code base but that's obviously out of scope for this ticket :)

Reply:
Don't forget; a lot of date handling should be pretty common, and probably handled by a dependency, so your maintenance effort is focused on the parts of our code that are novel to our app!

Reply:
Have modified the code to make it easier to understand.